app/src/main/res/xml/device_filter.xml

```diff
@@ -1,39 +1,41 @@
-    <usb-device vendor-id="1659" product-id="9187" /> <!-- 0x23e3: PL2303GE -->
-    <usb-device vendor-id="1659" product-id="9203" /> <!-- 0x23f3: PL2303GS -->
+    <!-- Prolific PL2303 Series (Vendor ID: 0x067B) -->
+    <usb-device vendor-id="1659" product-id="8963" />  <!-- 0x2303: PL2303HX, HXD, TA -->
+    <usb-device vendor-id="1659" product-id="9123" />  <!-- 0x23a3: PL2303GC -->
+    <usb-device vendor-id="1659" product-id="9139" />  <!-- 0x23b3: PL2303GB -->
+    <usb-device vendor-id="1659" product-id="9155" />  <!-- 0x23c3: PL2303GT -->
+    <usb-device vendor-id="1659" product-id="9171" />  <!-- 0x23d3: PL2303GL -->
+    <usb-device vendor-id="1659" product-id="9187" />  <!-- 0x23e3: PL2303GE -->
+    <usb-device vendor-id="1659" product-id="9203" />  <!-- 0x23f3: PL2303GS -->
 
-    <!-- 0x1a86 / 0x?523: Qinheng CH34x -->
+    <!-- Qinheng CH34x USB to Serial Devices (Vendor ID: 0x1A86) -->
     <usb-device vendor-id="6790" product-id="21795" /> <!-- 0x5523: CH341A -->
     <usb-device vendor-id="6790" product-id="29987" /> <!-- 0x7523: CH340 -->
+    <usb-device vendor-id="6790" product-id="21972" /> <!-- 0x55D4: CH9102F -->
 
-    <!-- CDC driver -->
-    <usb-device vendor-id="1240" product-id="223"/> <!-- 1240 / 223: MCP2200(For PSLab V5) -->
-    <usb-device vendor-id="9025" />                   <!-- 0x2341 / ......: Arduino -->
-    <usb-device vendor-id="5824" product-id="1155" /> <!-- 0x16C0 / 0x0483: Teensyduino  -->
-    <usb-device vendor-id="1003" product-id="8260" /> <!-- 0x03EB / 0x2044: Atmel Lufa -->
-    <usb-device vendor-id="7855" product-id="4"    /> <!-- 0x1eaf / 0x0004: Leaflabs Maple -->
-    <usb-device vendor-id="3368" product-id="516"  /> <!-- 0x0d28 / 0x0204: ARM mbed -->
-    <usb-device vendor-id="1155" product-id="22336" /><!-- 0x0483 / 0x5740: ST CDC -->
-    <usb-device vendor-id="11914" product-id="5"   /> <!-- 0x2E8A / 0x0005: Raspberry Pi Pico Micropython -->
-    <usb-device vendor-id="11914" product-id="10"  /> <!-- 0x2E8A / 0x000A: Raspberry Pi Pico SDK -->
-    <usb-device vendor-id="6790" product-id="21972" /><!-- 0x1A86 / 0x55D4: Qinheng CH9102F -->
-</resources>
\ No newline at end of file
+    <!-- CDC (Communication Device Class) Devices -->
+    <usb-device vendor-id="1240" product-id="223"/>   <!-- MCP2200 (For PSLab V5) -->
+    <usb-device vendor-id="9025" />                   <!-- Arduino (Generic) -->
+    <usb-device vendor-id="5824" product-id="1155" /> <!-- Teensyduino -->
+    <usb-device vendor-id="1003" product-id="8260" /> <!-- Atmel LUFA -->
+    <usb-device vendor-id="7855" product-id="4"    /> <!-- Leaflabs Maple -->
+    <usb-device vendor-id="3368" product-id="516"  /> <!-- ARM mbed -->
+    <usb-device vendor-id="1155" product-id="22336" /> <!-- ST CDC (0x0483 / 0x5740) -->
+    
+    <!-- Raspberry Pi Pico -->
+    <usb-device vendor-id="11914" product-id="5"   /> <!-- Micropython -->
+    <usb-device vendor-id="11914" product-id="10"  /> <!-- Pico SDK -->
```

Comment:
@td15 Did you remove the hex representations of the IDs in the lines above on purpose or was it by mistake?